### PR TITLE
Server-side encryption

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/stolostron/backlog/issues/21888

### CODE REVIEW

File: clusters/backup_and_restore.adoc [hunk 1]
```diff
@@ -143,6 +144,25 @@ spec:
 
 Refer to the link:https://github.com/openshift/oadp-operator/blob/master/docs/config/resource_req_limits.md[Velero resource requests and limits customization] to find out more about the `DataProtectionApplication` parameters.
 
+[#protecting-data-using-server-side-encryption]
+=== Protecting data using server-side encryption
+
+Server-side encryption is data encryption for the application or service that receives the data at the storage location. The backup mechanism itself does not encrypt data while in-transit (as it travels to and from backup storage location), or at rest (while it is stored on disks at backup storage location). Instead it relies on the native mechanisms in the object and snapshot systems.
+
+You can use `serverSideEncryption` and `kmsKeyId` parameters to enable encryption for the backups stored in Amazon S3. For more details, see the link:link:https://github.com/vmware-tanzu/velero-plugin-for-aws/blob/main/backupstoragelocation.md[Backup Storage Location YAML]. The following sample specifies an AWS KMS key ID when setting up the `DataProtectionApplication` resource:
```
Comment (sahare) [inline clusters/backup_and_restore.adoc]: Remove the duplicate link from `link:link:https:`

**Review by sahare — CHANGES_REQUESTED** [review]
Thanks @dockerymick , just a small typo

File: clusters/backup_and_restore.adoc (hunk 1 above)
Comment (dockerymick) [reply clusters/backup_and_restore.adoc]: Great catch! Thanks!

**Comment by dockerymick** [comment]
Received confirmation to merge this PR

**Pull request merged**